lib/honeybadger/cli/install.rb

```diff
@@ -46,7 +46,28 @@ def run
           File.open(path, 'w+') do |file|
             file.write(<<-CONFIG)
 ---
+# For more options, see https://docs.honeybadger.io/lib/ruby/gem-reference/configuration
+
 api_key: '#{api_key}'
+
+# The environment your app is running in.
+env: #{defined?(::Rails.application) ? "<%= Rails.env %>" : "<%= ENV['RUBY_ENV'] || ENV['RACK_ENV'] %>"}
+
+# The absolute path to your project folder.
+root: <%= Dir.pwd %>
```

Comment:
This should be `Rails.root` in Rails projects. Also, do we want to wrap it with quotes, like the `env` option?